### ISSUE TITLE
Focus visible ring not being taken from `button.tsx`

### ISSUE THREAD

**Issue by lorikku** [Aslam97/react-confirm-dialog#6]
Right now the 'focus' ring is black

![image](https://github.com/user-attachments/assets/aaa02b61-af0a-4c61-97f2-0dcfc1e0037d)

But my buttons (and all other elements) should have a coloured (purple) focus ring like this

![image](https://github.com/user-attachments/assets/8cd76a59-3fc0-48b3-8cbe-36a337bde199)

This was set on the cva of my `button.tsx`

![image](https://github.com/user-attachments/assets/606307d7-d248-4c20-9ca2-159af5a724ed)

But it's not being used by your confirm dialog. Any way this could be fixed? Neat little package btw 👍

**Comment by jess232017** [comment]
Hi @lorikku,

Since the confirm dialog is built with ShadCN UI components, you can customize it to match your design system by directly modifying the component.

You can copy the source code from this [URL](https://github.com/Aslam97/react-confirm-dialog/blob/main/packages/confirm-dialog/src/confirm-dialog.tsx) and paste it into your project. From there:

Replace the default components with your ShadCN UI components.
Apply your cva styles or other customizations.
